public/header.ejs

```diff
@@ -15,20 +15,20 @@
     <meta name="twitter:image" content="/imgs/revuc@192.png"/>
     <meta name="twitter:image:alt" content="RevolutionUC Hex Logo"/>
 
+    <script>
```

Comment:
Keep this script block for now. This might be removed if we have the server handle redirects.